dashboard/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "dashboard",
 	"version": "0.0.1",
 	"scripts": {
-		"dev": "svelte-kit dev --host 0.0.0.0",
+		"dev": "svelte-kit dev --host 0.0.0.0 --port 3003",
```

Comment:
pls remove `--port 3003`. Next time you can simply run the dashboard like 
`npm run dev -- -p 3003`

Reply:
Good eyes and nice tip- thanks.